### DISCUSSION TITLE
Conditional fields based in Multi Select field values

### DISCUSSION THREAD

**Discussion by pazunino** [area17/twill#2432] [Ideas]
## Summary

Add support for Multi Select field in Conditional fields

## Describe the solution you'd like

When you add a checked option on the Multi Select a related Conditional field is activated based on the value of the checked option.
Could be fine if the content of the conditional fields is saved in a pivot column.

**Comment by ifox** [comment]
I'm not sure about the use case for this, and why it would save the conditional fields in a pivot. If you're looking to select multiple related records and add pivot fields, I'd suggest looking into the inline repeater field and it's browser option in Twill 3. In Twill 2, you can probably accomplish this with a repeater field hosting a single select and additional fields.
